test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -475,6 +482,12 @@ func (bp *BindPlugin) Bind(ctx context.Context, state *framework.CycleState, p *
 
 	bp.numBindCalled++
 	if bp.pluginInvokeEventChan != nil {
+		defer func() {
+			if r := recover(); r != nil {
+				klog.FromContext(ctx).Error(nil, "Recovered from panic in Bind", "error", r)
+				return
+			}
+		}()
 		bp.pluginInvokeEventChan <- pluginInvokeEvent{pluginName: bp.Name(), val: bp.numBindCalled}
```

Comment:
Isn't it sufficient to use `select` with `ctx.Done()` instead?
```suggestion
		select {
		case <-ctx.Done():
		case bp.pluginInvokeEventChan <- pluginInvokeEvent{pluginName: bp.Name(), val: bp.numBindCalled}:
		}
```